changelog/changes/single-awake-poster.md

```diff
@@ -0,0 +1,11 @@
+---
+title: "More supported types in `read_parquet"
+type: feature
+authors: IyeOnline
+pr: 5373
+---
+
+We have enabled the `read_parquet` operator to accept more types in the parquet
+files. It will convert integer, floating point an time types to the appropriate
```

Comment:
```suggestion
files. It will convert integer, floating point and time types to the appropriate
```

Please just make an AI grammar pass over changelogs and docs (doesn't mean that AI has to write it, but just let it fix issues like that)